network_listview_builder/lib/reloadListView.dart

```diff
@@ -0,0 +1,72 @@
+import 'package:flutter/material.dart';
+import 'package:network_listview_builder/pokemonItem.dart';
+import 'package:network_listview_builder/pokemonModel.dart';
+import 'package:network_listview_builder/pokemonNotifier.dart';
+
+class ReloadListView extends StatefulWidget {
+  const ReloadListView({Key key}) : super(key: key);
+
+  @override
+  _ReloadListViewState createState() => _ReloadListViewState();
+}
+
+class _ReloadListViewState extends State<ReloadListView> {
+  PokemonNotifier notifier;
+
+  @override
+  void initState() {
+    super.initState();
+    notifier = PokemonNotifier();
+    notifier.getMore();
+  }
+
+  @override
+  void dispose() {
+    notifier.dispose();
+    super.dispose();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return ValueListenableBuilder<List<Pokemon>>(
+        valueListenable: notifier,
+        builder: (BuildContext context, List<Pokemon> value, Widget child) {
+          return value != null
+              ? RefreshIndicator(
+                  onRefresh: () async {
+                    return await notifier.reload();
+                  },
+                  child: value.isEmpty
+                      ? ListView.builder(
```

Comment:
Do you need a listview here, if there is only one child?

Reply:
Yea need it so we can scroll up for reload